src/blocks/progress-bar/attributes.js

```diff
@@ -0,0 +1,42 @@
+const attributes = {
+	height: {
+		type: 'number',
+		default: 35
```

Comment:
Don't need a default. The default can be from the CSS.

Reply:
I can't do that, ResizableBox depends on it.

Reply:
Got it.